include/umf/mempolicy.h

```diff
@@ -25,9 +25,22 @@ typedef enum umf_mempolicy_membind_t {
     /// Bind memory to namespace
     UMF_MEMPOLICY_BIND,
     /// Prefer memory from namespace but fallback to other memory if not available
-    UMF_MEMPOLICY_PREFERRED
+    UMF_MEMPOLICY_PREFERRED,
+    /// Allocation will be split evenly across nodes specified in nodemask.
+    /// umf_mempolicy_split_partition_t can be used to specify different distribution
```

Comment:
`different distribution to specify other distribution`...? re-phrase I guess...?

Reply:
Done